configs/ocr_recognition_light_model.yaml

```diff
@@ -0,0 +1,43 @@
+# Example configuration for training a predefined light OCR recognition model
+
+model:
+  name: ocr_recognition_light
+  predefined_model:
+    name: OCRRecognitionModel
+    params:
+      variant: light
+      alphabet: ['a', 'b', 'c', 'd', 'e', 'h', 'l', 'm', 'n', 'o', 'p', 'x', 'y', 'z']
```

Comment:
We could have a set of predefined alphabets, so users don't have to enumerate all the letters. Something like `"ascii"`, `"numbers"`, `"alphanum"`, or `"english"` (and possibly some other languages)